packages/curvenote-cli/src/session/session.ts

```diff
@@ -87,12 +87,12 @@ export class Session implements ISession {
     this.PRIVATE_CDN = 'https://prv.curvenote.com';
     this.TEMP_CDN = 'https://tmp.curvenote.com';
     this.PUBLIC_CDN = 'https://cdn.curvenote.com';
-    if (url?.startsWith(STAGING_API_URL)) {
+    if (this.API_URL?.startsWith(STAGING_API_URL)) {
```

Comment:
@stevejpurves - is this change ok? It seems like we should make this selection based on the resolved `API_URL` not the token's `url`.